### ISSUE TITLE
Error with incompatible interface after installing

### ISSUE THREAD

**Issue by weaverryan** [netgen-layouts/layouts-sylius#9]
Hi!

After running (per the docs)

```
composer create-project netgen/layouts-sylius-site my_project
```

The app explodes with:

> Fatal error: Declaration of Netgen\Layouts\Sylius\Doctrine\ORM\ProductRepositoryInterface::findByTaxon(Sylius\Component\Channel\Model\ChannelInterface $channel, ?Sylius\Component\Taxonomy\Model\TaxonInterface $taxon, string $locale, int $offset, int $count, array $sorting = []): array must be compatible with Sylius\Component\Core\Repository\ProductRepositoryInterface::findByTaxon(Sylius\Component\Core\Model\TaxonInterface $taxon): array in /Users/weaverryan/Sites/knp/knpu-repos/practice/netgen_layouts_sylius/vendor/netgen/layouts-sylius/lib/Doctrine/ORM/ProductRepositoryInterface.php on line 18

Cheers!

**Comment by emodric** [comment]
Hi @weaverryan , this is due to project not supporting latest version of Sylius yet (1.12?).

In any case, thanks for the report, I'll look into it.

**Comment by emodric** [comment]
@weaverryan This is now fixed in `netgen/layouts-sylius` 1.3.2.

It turns out that Sylius added a new method to an interface we extend that has a same name as a method we added to our variant of the interface.

I renamed our methods and all is good again.
